README.md

```diff
@@ -2,8 +2,32 @@
 
 The source for the Ember.js guides.
 
+
 ### Contributing
 
+The Ember Guides are maintained and updated by an all-volunteer group of Ember community members. We'd love to have you join our efforts! 
+
+Please note that no attempt is made to update content, layout, or styles for older versions of the Guides. They are considered static and immutable, as it is too difficult to maintain content for every version ever released. Issues will only be fixed for future releases.
+
+## Fixing problems
+
+If you find a problem on a particular page in the Guides, the most helpful thing you can do is open a pull request. If you're not sure how to fix it, open an issue.
+
+## Contributing solicited content
+
+We try to make it easy for people to contribute to the Guides by tagging issues with _Help Wanted_ when appropriate. The best way to get started contributing content is to pick up one of these issues.
```

Comment:
maybe you could make `help wanted` a link to https://github.com/emberjs/guides/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22